src/Stryker.Core/Stryker.Core/Mutators/ObjectCreationMutator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Stryker.Core.Mutators
+{
+    public class ObjectCreationMutator : MutatorBase<ObjectCreationExpressionSyntax>, IMutator
```

Comment:
Since you did add this, it would probably be a good idea to describe this in the docs.

Reply:
It is described 👀